live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/messaging/transform/request/model/ATSTransformRequest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.hxi_connector.live_ingester.messaging.transform.request.model;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import lombok.Builder;
+import lombok.ToString;
+
+/**
+ * Model used for Transform Request Events.
+ *
+ * This is a mirror of org.alfresco.repo.rendition2.TransformRequest in Alfresco Repository project.
+ */
+@Builder
+public record ATSTransformRequest(String requestId,
+                                  String nodeRef,
+                                  String targetMediaType,
+                                  String clientData,
+                                  Map<String, String> transformOptions,
+                                  String replyQueue) implements Serializable
```

Comment:
I think we should keep this model similar to the original class because it reduces our code (and what needs unit testing). This will be tested by the E2Es. I think it doesn't matter too much because it's meant to be the external interface. In fact better would be if we could import this class as a dependency.